packages/gatsby/src/utils/page-ssr-module/lambda.ts

```diff
@@ -148,8 +154,8 @@ async function getEngine(): Promise<GraphQLEngineType> {
         reject(error)
       })
     })
+    console.log(`Downloaded datastore from CDN`)
   }
-  console.log(`Downloaded datastore from CDN`)
```

Comment:
This is unrelated to rest of changes, just noticed this was logged even if LMDB wasn't excluded from the bundle (so when it shouldn't be logged)